src/test/java/org/jenkinsci/plugins/kubernetes/credentials/OpenShiftBearerTokenCredentialBindingTest.java

```diff
@@ -0,0 +1,109 @@
+package org.jenkinsci.plugins.kubernetes.credentials;
+
+import com.cloudbees.plugins.credentials.CredentialsProvider;
+import com.cloudbees.plugins.credentials.CredentialsScope;
+import com.cloudbees.plugins.credentials.domains.Domain;
+import hudson.FilePath;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.servlet.ServletContextHandler;
+import org.eclipse.jetty.servlet.ServletHolder;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.jenkinsci.plugins.workflow.test.steps.SemaphoreStep;
+import org.junit.*;
+import org.junit.runners.model.Statement;
+import org.jvnet.hudson.test.BuildWatcher;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+import java.io.IOException;
+import java.util.Collections;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.codehaus.groovy.runtime.DefaultGroovyMethods.grep;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class OpenShiftBearerTokenCredentialBindingTest {
+
+    @Rule
```

Comment:
I don't know anything about `RestartableJenkinsRule`, but I believe it replaces the `JenkinsRule` and therefore, this one can be removed